api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -192,6 +201,38 @@ message SendSatelliteCommandsRequest {
   repeated bytes command = 2;
 }
 
+// A request to modify configuration of ground station hardware.
+//
+// Next ID: 2
+// Status: ALPHA This API is under development and may not work correctly or be changed in backwards
+//         incompatible ways in the future.
+message GroundStationConfigurationRequest {
+  // A request to modify transmitter configuration at a ground station.
+  TransmitterConfigurationRequest transmitter_configuration_request = 1;
+}
+
+// A request to configure a transmitter at a ground station. Default state of these parameters
+// depends on hardware configuration at each ground station. Please check TransmitterConfiguration
+// in monitoring message. (Currently, some parameters are not supported.)
```

Comment:
> (Currently, some parameters are not supported.)

I think you can leave this out since you added the `ALPHA` status below.

Reply:
Also, I think it's okay to not have the `ALPHA` message here because you have `ALPHA` status on `ground_station_configuration_request` in `SatelliteStreamRequest`.